api/src/data/migrations/015.create-incident-types.ts

```diff
@@ -6,12 +6,12 @@ export async function up(knex: knex.Knex) {
         table.string("name", 256).notNullable();
         table.string("description", 4096).nullable();
         table.boolean("searchable").notNullable();
-        table.integer("added_by").notNullable(); // look into renaming this
```

Comment:
Can you make this `added_by_role_id`? I don't think it's obvious that it refers to a role, unless explicitly stated.